spec/controllers/requests_controller_spec.rb

```diff
@@ -61,6 +61,21 @@
       end
     end
 
+    describe 'Illiad requests' do
```

Comment:
I don't think you need this test, because you're combining all the request types in the patron model.

Reply:
I added this b/c they had a similar BorrowDirectRequests construct above. I can remove it if not necessary or relevant.

Reply:
Yeah, now I see it. Seems like an odd test to me. It's testing whether the thing you assigned in the test is assigned.

Reply:
I can remove it